tools/README.md

```diff
@@ -7,6 +7,7 @@ Rscript tools/yarn_install.R
 ```
 
 **Warning:** Upgrading to a new major version of Bootstrap won't be as simple as running this R script. We'll cross that bridge when we get there.
+Note that after upgrading Bootstrap, you should generate the precompiled CSS files by running the precompile-css.R script in this directory. (Eventually this step should be added to the Bootstrap upgrade script.)
```

Comment:
Is there any reason why we can't do this at the end of the `tools/yarn_install.R` script (instead of having a separate build script)? I'm worried we'll forget to run this and wind up shipping stale versions of Bootstrap

Reply:
I didn't add it there because the comment said "Upgrading to a new major version of Bootstrap won't be as simple as running this R script", so I assumed that it wasn't safe to put there. But if you think it can go there, I'm fine with that. Note that the package needs to be loaded to run the script.

FWIW, the unit tests ensure that the precompiled css matches the normally compiled css. I tested this out by modifying one of the scss files and running the unit tests. The tests failed, which is good.

Reply:
Ok, great that we have the test...I think I'd still prefer that we move the logic to `tools/yarn_install.R`...We could even loop over the `versions` as I mentioned in https://github.com/rstudio/bootstraplib/pull/109/files#r491023391

Reply:
It's not clear to me exactly what yarn_install.R does with respect to Bootstrap. When we want to update Bootstrap, do we need to manually update Bootstrap first, and then run that script? If so, then I agree that the precompilation stuff can go there.

But if we run the script and then upgrade Bootstrap, then it doesn't make sense to precompile in that script.

Reply:
> It's not clear to me exactly what yarn_install.R does with respect to Bootstrap.

It runs `yarn install` with https://github.com/rstudio/bootstraplib/blob/master/inst/package.json, then a bunch of post-processing on the generated files

> do we need to manually update Bootstrap first, and then run that script?

Yea, you currently need to modify the package.json to update or remove the lock file